lib/KTable/KTableGridItem.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+
+  <td
+    :class="$computedClass(coreOutlineFocus)"
+    :style="{ textAlign: textAlign, minWidth: minWidth, width: width }"
+    tabindex="0"
+    role="gridcell"
+    @keydown="onKeydown"
+  >
+    <slot :content="content">
+      {{ content }}
+    </slot>
+  </td>
+
+</template>
+
+
+<script>
+
+  import { DATA_TYPE_NUMERIC } from '../composables/useSorting';
+
+  export default {
+    name: 'KTableGridItem',
+    props: {
+      content: {
+        type: [String, Number],
+        required: true,
+      },
+      dataType: {
+        type: String,
+        default: 'string',
+      },
+      rowIndex: {
+        type: Number,
+        required: true,
+      },
+      colIndex: {
+        type: Number,
+        required: true,
+      },
+      minWidth: {
+        type: String,
+        default: 'auto',
+      },
+      width: {
+        type: String,
+        default: 'auto',
+      },
+    },
+    computed: {
+      coreOutlineFocus() {
+        return {
+          ':focus': {
+            ...this.$coreOutline,
+            outlineOffset: '-2px',
+          },
+        };
+      },
+      textAlign() {
+        return this.dataType === DATA_TYPE_NUMERIC ? 'right' : 'left';
```

Comment:
```suggestion
       const alignLtr = this.dataType === DATA_TYPE_NUMERIC ? 'right' : 'left';
       if (this.isRtl && alignLtr === 'right') { return 'left' }
       if  (this.isRtl && alignLtr === 'left') {return 'right'}
       return alignLtr;
```

or something like that..

**[required] Ensures that in right-to-left (RTL) languages, text alignment will be the opposite.**

RTLSCSS framework takes care automatically for anything in `<styles>` but for these dynamic JavaScript styles, we need to use `isRtl` utility. Details in [Kolibri developer documentation > RTL language support](https://kolibri-dev.readthedocs.io/en/develop/i18n.html#rtl-language-support).